src/sentry/snuba/outcomes.py

```diff
@@ -320,8 +320,13 @@ def run_outcomes_query_totals(
 
 
 def run_outcomes_query_timeseries(
-    query: QueryDefinition, tenant_ids: dict[str, Any] | None = None
+    query: QueryDefinition, tenant_ids: dict[str, Any] | None = None, referrer="outcomes.timeseries"
 ) -> ResultSet:
+    """
+    Runs an outcomes query. By default the referrer is `outcomes.timeseries` and this should not change
+    unless there is a very specific reason to do so. Eg. getsentry uses this function for billing
+    metrics, so the referrer is different as it's no longer a "product" query.
+    """
```

Comment:
I can PR this separately but it's a quick change not very relevant to the entire PR. Previously, I added `tenant_ids` to all `outcomes.timeseries` and `outcomes.totals` queries but I was still seeing timeseries queries coming without this data. Turns out this function is used in getsentry for a billing metric, and there should should really be a different referrer for that use case, so I made referrer an optional param with a quick description of why.